python/oneflow/test/modules/test_functional_docstr.py

```diff
@@ -51,7 +51,7 @@ def _run_functional_doctest(
 class TestFunctionalDocstrModule(flow.unittest.TestCase):
     def test_functional_docstr(test_case):
         arg_dict = OrderedDict()
-        arg_dict["module"] = [flow, flow.Tensor]
+        arg_dict["module"] = [flow, flow.sbp, flow.env, flow.Tensor]
```

Comment:
```suggestion
        arg_dict["module"] = [flow, flow.sbp, flow.env]
```

这个 PR 的改动，是不是不涉及到 `flow.Tensor` 下测不了的 docstr。有另外一个专门的 test_tensor.py 文件专门测 Tensor 下的方法。